### DISCUSSION TITLE
Edit - issue Solved

### DISCUSSION THREAD

**Discussion by prashantjagtap2909** [community/community#140710] [Other Feature Feedback, Questions, & Ideas]
Edited - Issue is solved

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @prashantjagtap2909 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
